README.md

```diff
@@ -12,7 +12,7 @@ For more information, see https://bisq.network/intro and for step-by-step gettin
 
 ## Building Bisq
 
-You will need OpenJDK [JDK 10](https://jdk.java.net/10/) installed to complete the following instructions.
+You will need either [Oracle JDK 11](https://www.oracle.com/technetwork/java/javase/downloads/jdk11-downloads-5066655.html) or [OpenJDK JDK 11](https://jdk.java.net/11/) installed to complete the following instructions.
```

Comment:
Same as above; let's just keep it to OpenJDK 11. Note that there's a typo (not introduced by this PR) here: `Open JDK JDK 11` should just be `OpenJDK 11`.